README.md

```diff
@@ -615,9 +615,11 @@ The Onfido SDK has been optimised to provide the following accessibility support
 
 ### Browser compatibility
 
-![Chrome](https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_48x48.png) | ![Firefox](https://raw.githubusercontent.com/alrra/browser-logos/master/src/firefox/firefox_48x48.png) | ![IE](https://raw.githubusercontent.com/alrra/browser-logos/master/src/edge/edge_48x48.png) | ![Safari](https://raw.githubusercontent.com/alrra/browser-logos/master/src/safari/safari_48x48.png)
---- | --- | --- | --- |
-Latest ✔ | Latest ✔ | 11+ ✔ | Latest ✔ |
+![Chrome](https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_48x48.png) | ![Firefox](https://raw.githubusercontent.com/alrra/browser-logos/master/src/firefox/firefox_48x48.png) | ![IE](https://raw.githubusercontent.com/alrra/browser-logos/master/src/archive/internet-explorer_9-11/internet-explorer_9-11_48x48.png) | ![Edge](https://raw.githubusercontent.com/alrra/browser-logos/master/src/edge/edge_48x48.png) | ![Safari](https://raw.githubusercontent.com/alrra/browser-logos/master/src/safari/safari_48x48.png)
+--- | --- | --- | --- | --- |
+Latest ✔ | Latest * ✔ | 11 ✔ | Latest ✔ | Latest ✔ |
+\* _Firefox on Android, iOS not supported_
```

Comment:
currently this becomes a new row in the table, which looks a bit odd (as it becomes part of the chrome column). Can this sit outside the table?

Reply:
Ah, yes. Oddly it didn't appear that way in my IDE's preview. Will make it sit outside the table as I'd intended. Thanks for spotting!

Reply:
no problem it looks great now. thanks